### PR TITLE
fsevents: i18n improvements for mac file paths

### PR DESCRIPTION
**Pull request by chrisbank**
Fix a bug monitoring folders with japanese characters in the path.

### CODE REVIEW

File: src/unix/fsevents.c [hunk 1]
```diff
@@ -339,9 +339,8 @@ static void uv__fsevents_reschedule(uv_fs_event_t* handle) {
       curr = ngx_queue_data(q, uv_fs_event_t, cf_member);
 
       assert(curr->realpath != NULL);
-      paths[i] = CFStringCreateWithCString(NULL,
-                                           curr->realpath,
-                                           CFStringGetSystemEncoding());
+      paths[i] = CFStringCreateWithFileSystemRepresentation(NULL,
+                                            curr->realpath);
```
Comment (indutny) [inline src/unix/fsevents.c]: Can you either line it up with `NULL` on previous line, or put `NULL` on separate line too (after 4 space indent)?

**Comment by indutny** [comment]
LGTM, except very minor style nit!

**Comment by indutny** [comment]
And thanks for contributing it! ;) But please sign CLA, so I'd be able to pull it.

**Comment by indutny** [comment]
@chrisbank ping?

**Comment by chrisbank** [comment]
Sorry, I'm trying to get the CLA signed and should have realized it would take a bit of time for my work to approve it. Just waiting on an email back from my manager to say it's ok then I'll sign the CLA and push the whitespace change too,

**Comment by indutny** [comment]
Ah, sure! Thanks for contributing anyway, looking forward for continuing here.

**Comment by chrisbank** [comment]
Got the thumbs up, should be good to go.

**Comment by indutny** [comment]
Thank you, landed in v0.10 branch in 3780e12. Bonus points if you'll implement test for it ;)

**Pull request closed without merging**